functional/ek-cert-use-ek_handle-custom-ca_certs/test.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+AGENT_ID="d432fbb3-d2f1-4a97-9ef7-75bd81c00000"
+CERTDIR=/var/lib/keylime/tpm_cert_store
+CACERTDIR=/var/lib/swtpm-localca
+PASSWORD_TPM=abc123
+
+rlJournalStart
+
+    rlPhaseStartSetup "Do the keylime setup"
+        rlRun 'rlImport "./test-helpers"' || rlDie "cannot import keylime-tests/test-helpers library"
+        rlAssertRpm keylime
+        # update /etc/keylime.conf
+        limeBackupConfig       
+        # if TPM emulator is present
+        if rlIsRHEL '<9'; then
+            rlDie "Swtpm is only on RHEL-9 and above."
+        fi
+        if limeTPMEmulated; then
+            # start tpm emulator
+            rlRun "limeStartTPMEmulator"
+            rlRun "limeWaitForTPMEmulator"
+            # make sure tpm2-abrmd is running
+            rlServiceStart tpm2-abrmd
+            sleep 5
+            # start ima emulator
+            rlRun "limeInstallIMAConfig"
+            rlRun "limeStartIMAEmulator"
+        else
+            rlDie "Test work only when TPM is emulated."
+        fi
+        #change tpm_ownerpassword    
+        rlRun "limeUpdateConf cloud_agent tpm_ownerpassword $PASSWORD_TPM"
```

Comment:
Please add this line below line 48. Also, better use comment "Configure TPM owner password", we are not really changing it here.